src-version-changes/changed-in-ghc96/ghc94/Data/Tuple/Linear/Compat.hs

```diff
@@ -0,0 +1,16 @@
+{-# LANGUAGE LinearTypes #-}
+module Data.Tuple.Linear.Compat where
```

Comment:
I'm pondering whether I want `Data.Tuple.Linear.Compat` to be hidden in Haddock as an internal module (applies to both copies).